src/components/Post/Comments/Comments.tsx

```diff
@@ -1,14 +1,12 @@
-// @flow strict
-import React from 'react';
-import ReactDisqusComments from 'react-disqus-comments';
+import React from 'react';import ReactDisqusComments from 'react-disqus-comments';
```

Comment:
Codacy has a fix for the issue: [Missing whitespace after semicolon.](https://app.codacy.com/gh/alxshelepenok/gatsby-starter-lumen/pullRequest?prid=7771704)
```suggestion
import React from 'react'; import ReactDisqusComments from 'react-disqus-comments';
```